### PR TITLE
Update grammar

### PR DESCRIPTION
**Pull request by Snipeur060**
## PR Checklist

**Tests and Styling**
- [x] Has pytest style unit tests (and `pytest` passes).
- [x] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [x] New features are documented, with examples if plot related.
- [x] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [x] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [x] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: doc/users/explain/interactive.rst [hunk 1]
```diff
@@ -170,7 +170,7 @@ and run the GUI event loop for the specified period of time.
 The GUI event loop being integrated with your command prompt and
 the figures being in interactive mode are independent of each other.
 If you use `pyplot.ion` but have not arranged for the event loop integration,
-your figures will appear but will not be interactive while the prompt is waiting for input.
+your figures will appear, but will not be interactive while the prompt is waiting for input.
```
Comment (tacaswell) [inline doc/users/explain/interactive.rst:173]:
```suggestion
your figures will appear but will not be interactive while the prompt is waiting for input.
```

I do not think there should not be a comma here.

**Comment by tacaswell** [comment]
Thank you for your effort @Snipeur060 .  However I think one of these changes is wrong and the other two are neutral (neither improve nor harm the docs).

File: doc/users/explain/interactive.rst (hunk 1 above)
Comment (jklymak) [reply doc/users/explain/interactive.rst:173]: I agree the comma is wrong here.   On the other hand, I think a sentence with more than one "but" is confusing, maybe: 
```suggestion
If you try to use `pyplot.ion` without arranging for the event-loop integration,
your figures will appear but will not be interactive while the prompt is waiting for input.
```

Comment (Snipeur060) [reply doc/users/explain/interactive.rst:173]: I will change I think is better

**Comment by Snipeur060** [comment]
i change in recommandation of @jklymak (thx)

**Comment by Snipeur060** [comment]
if it's good for everyone it's good for me :-)

**Comment by jklymak** [comment]
Can you squash to one commit, or would you like us to do it?

**Review by jklymak — APPROVED** [review]

**Comment by Snipeur060** [comment]
sorry i don't know how :-(

**Comment by jklymak** [comment]
We can do it for you if you prefer, but if you want to try (backup your branch first) https://stackoverflow.com/questions/14534397/squash-all-my-commits-into-one-for-github-pull-request.

**Comment by Snipeur060** [comment]
i will try

**Comment by Snipeur060** [comment]
ok i juste understand the version where you squash and merge but the other i don't understand

**Comment by Snipeur060** [comment]
(I prefer that you do)

**Comment by tacaswell** [comment]
Thank you for you work on this @Snipeur060 and congratulations on your first merged Matplotlib PR 🎉 .  Hope we hear from you again!

**Pull request merged**